src/main/java/bio/terra/workspace/service/workspace/flight/FlightMapKey.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.workspace.service.workspace.flight;
+
+public interface FlightMapKey {
```

Comment:
Where is this interface used/going to be used?

Reply:
It's not anymore. Deleted. It was almost a good idea, but generics aren't quite smart enough for it to work.